src/cli/migrate/badge-variant-enum.ts

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2020, SumUp Ltd.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Transform } from 'jscodeshift';
+
+import { renameJSXAttribute, findLocalNames } from './utils';
+
+const transform: Transform = (file, api) => {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  const components = findLocalNames(j, root, 'Badge');
+
+  if (!components) {
+    return;
+  }
+
+  components.forEach(component => {
+    renameJSXAttribute(j, root, component, 'color', 'variant');
+  });
+
+  return root.toSource();
+};
+
+export default transform;
```

Comment:
Can you explain why we need to do this? 🙂

Reply:
The migration guide has more context on codemods: https://github.com/sumup-oss/circuit-ui/blob/beta/MIGRATION.md#-codemods

TL;DR: Codemods enable users to apply breaking changes automatically.

Reply:
That's amazing... 💯 👍 
I'm impressed 🙂 😉